src/dash_app.py

```diff
@@ -229,20 +86,7 @@ def show_instructions(n_clicks):
     if n_clicks is None or n_clicks % 2 == 0:
         return ''
     else:
-        return """
-        SAEMI is a tool for obtaining a Size Analysis of particles
-        in Electron Microscopy Images. To obtain a size analysis,
-        first upload an Electron Microscopy image using the Upload
-        button below. The tool will then predict how to segment the
-        particles from the background using a model trained through
-        deep learning. After segmentation, the tool will label each
-        separate segment and count each segment's number of pixels.
-        This size distribution is then displayed in a histogram which
-        can be used to determine properties such as the mean size,
-        median size or standard deviation of sizes in the image.
-        Thank you for considering using this tool and any additional
-        feedback is always welcome. Good luck!
-        """
+        return dl.instructions()
```

Comment:
Good thing you refactored this. Having instructions hard coded in like this is usually a bad idea.